src/control/system/member.go

```diff
@@ -161,6 +162,14 @@ func (ms MemberState) isTransitionIllegal(to MemberState) bool {
 			MemberStateReady:  true,
 			MemberStateJoined: true,
 		},
+		MemberStateExcluded: {
+			MemberStateErrored:      true,
+			MemberStateStopping:     true,
+			MemberStateStopped:      true,
+			MemberStateStarting:     true,
+			MemberStateReady:        true,
+			MemberStateUnresponsive: true,
+		},
```

Comment:
Since @tanabarr has removed some of these in 2023, I think we can benefit from knowing whether that was for fixing an issue of just "user-friendly" messages or an issue of functionality.

Reply:
Hmm, fair point. I didn't realize the history. It looks like the original issue was functional in nature, and maybe we're being a bit too strict internally about what we end up reporting to the user. I'll investigate this more tomorrow.

Reply:
Sure, thanks. I've just approved for now.

Reply:
Thinking about state transitions in another context, I think it's helpful for admins to know when a Rank is Stopped as opposed to Excluded. For example if we don't allow Excluded->Stopped then the user can't tell whether the relevant engine process is running. I would be in favour of allowing that transition and ensuring that rejoin suicides AdminExcluded ranks.